cli/commands/server/start.go

```diff
@@ -106,6 +107,10 @@ custom: allow pruning options to be manually specified through 'pruning-keep-rec
 		RunE: func(cmd *cobra.Command, _ []string) error {
 			logger := clicontext.GetLoggerFromCmd[LoggerT](cmd)
 			cfg := clicontext.GetConfigFromCmd(cmd)
+			if cfg.Consensus.TimeoutCommit == 0 {
+				return errors.New("please edit your config.toml file and set timeout_commit to 1s")
+			}
+
```

Comment:
What was the motivation for adding this check?

Reply:
>  @abi87 What was the motivation for adding this check? Is this related to the dep updated or just wanting to  prevent footguns and ensure >=1s block times?

Immediate tests following dep updates showed a large number of consensus blocks produced pretty rapidly. We realized some of configs were not properly handled (wrongly picking cometBFT default values instead of BeaconKit) and we added this check as a fail fast kind of solution

Reply:
Thank you for the context - this is very helpful!